src/v/cluster/scheduling/partition_allocator.cc

```diff
@@ -226,8 +228,11 @@ std::error_code partition_allocator::check_cluster_limits(
         if (memory_limit > 0 && proposed_total_partitions > memory_limit) {
             vlog(
               clusterlog.warn,
-              "Refusing to create {} partitions, exceeds memory limit {}",
+              "Refusing to create {} new partitions as total partition count "
+              "{} "
+              "would exceed memory limit {}",
```

Comment:
I found this one and the below always confusing as it kind of compares different units that don't really make sense without actually reading the code of how they translate to each other.